UI/window-basic-auto-config.cpp

```diff
@@ -361,6 +361,11 @@ bool AutoConfigStreamPage::validatePage()
 	if (!ui->doBandwidthTest->isChecked()) {
 		bitrate = ui->bitrate->value();
 		wiz->idealBitrate = bitrate;
+#if YOUTUBE_ENABLED
+	} else if (IsYouTubeService(wiz->serviceName)) {
+		/* Set maximum to YouTube limit */
+		bitrate = 51000;
+#endif
 	}
```

Comment:
I feel like this was supposed to replace the 10000kbps cap for Twitch with 51000kbps when running the bandwidth test on Youtube, but that isn't what's happening - it will set the user's bitrate to 51000 if the bandwidth test is not selected, ignoring the spinbox.

Reply:
The `else` here will only get executed if `doBandwidthTest` is checked so this code shouldn't cause the issue you're describing.

Reply:
I've now tested this with and without an account being connected and it works correctly.

Reply:
Oops, that's what I get for reviewing at 2am. I still think it's an unclear code pattern, moving the 10000 assignment in there might help.

Reply:
I will adjust the code here a bit, and also set the upper bound for YouTube based on the resolution rather than just using the max all the time.